ddp-workspace/projects/ddp-dsm-ui/src/app/FON/fon.module.ts

```diff
@@ -0,0 +1,81 @@
+import {NgModule} from '@angular/core';
+import {NavigationComponent} from './layout/navigation/navigation.component';
+import {FonComponent} from './fon.component';
+import {ActivityComponent} from './pages/activities/activity/activity.component';
+import {ParticipantsListComponent} from './pages/participantsList/participantsList.component';
+import {ActivitiesComponent} from './pages/activities/activities.component';
+import {HomeComponent} from './pages/home/home.component';
+import {fonRoutingModule} from './fon-routing.module';
+import {CommonModule} from '@angular/common';
+import {DdpModule, SessionMementoService} from 'ddp-sdk';
+import {MatExpansionModule} from '@angular/material/expansion';
+import {MatDividerModule} from '@angular/material/divider';
+import {MatListModule} from '@angular/material/list';
+import {MatIconModule} from '@angular/material/icon';
+import {MatPaginatorModule} from '@angular/material/paginator';
+import {MatProgressSpinnerModule} from '@angular/material/progress-spinner';
+import {MatProgressBarModule} from '@angular/material/progress-bar';
+import {MatButtonModule} from '@angular/material/button';
+import {StoreService} from '../STORE/store.service';
+import {MainConstants} from './constants/main-constants';
+import {TranslateService} from '@ngx-translate/core';
+import {SessionService} from '../services/session.service';
+import {Title} from '@angular/platform-browser';
+
+
+const material = [
+  MatExpansionModule,
+  MatDividerModule,
+  MatListModule,
+  MatIconModule,
+  MatPaginatorModule,
+  MatProgressSpinnerModule,
+  MatProgressBarModule,
+  MatButtonModule
+];
+
+@NgModule({
+  declarations: [
+    NavigationComponent,
+    FonComponent,
+    ActivityComponent,
+    ParticipantsListComponent,
+    ActivitiesComponent,
+    HomeComponent
+  ],
+  imports: [
+    CommonModule,
+    fonRoutingModule,
+    DdpModule.forDSM(),
+    ...material,
+  ],
+  providers: [],
+  exports: []
+})
+
+export class fonModule {
+  constructor(private storeService: StoreService,
+              private translateService: TranslateService,
+              private sessionService: SessionService,
+              private dssSessionService: SessionMementoService,
+              private title: Title) {
+
```

Comment:
Fon module is used for initializing important configurations, like setting a study name in STORE, translations, DSM token into DSS and etc.